bin/plotting/pycbc_plot_dq_flag_likelihood

```diff
@@ -76,7 +90,10 @@ ax2.set_ylabel('DQ Log Likelihood Penalty')
 ax2.set_ylim(numpy.log(ymin), numpy.log(ymax))
 
 # add meta data and save figure
-plot_title = f'{ifo}:{args.dq_label} DQ Trigger Rates'
+if args.title:
```

Comment:
Use `if args.title is not None` - If someone wanted to use an empty string as title, this would not do it right